babel.config.mjs

```diff
@@ -53,7 +52,7 @@ export default function getBabelConfig(api) {
     [
       '@mui/internal-babel-plugin-minify-errors',
       {
-        missingError,
+        missingError: 'annotate',
```

Comment:
We could consider `'throw'` here. I believe the idea behind `'annotate'` is that in CI you run a search on built files for these annotations, but we don't do that. We can try this later though.